engine-tests/src/test/java/org/terasology/fixtures/TestStorageManager.java

```diff
@@ -33,7 +32,7 @@ public TestStorageManager(Map<Vector3ic, ChunkStore> chunkStores) {
     }
 
     public void add(Chunk chunk) {
-        chunkStores.put(chunk.getPosition(new Vector3i()), new TestChunkStore(chunk));
+        chunkStores.put(chunk.getPosition(), new TestChunkStore(chunk));
```

Comment:
Hm, we often recommend to create a copy when storing vectors in a map, but I guess for chunk position it is fine to assume the position to be constant and stable for a chunk, not changing ever. :thinking: